src/luascript.cpp

```diff
@@ -10931,6 +10892,35 @@ int LuaScriptInterface::luaPlayerSendCreatureSquare(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaPlayerSendWorldLight(lua_State* L)
+{
+	// player:sendWorldLight(color, level)
+	Player* player = getUserdata<Player>(L, 1);
+	if (player) {
+		LightInfo light;
+		light.color = getNumber<uint8_t>(L, 2);
+		light.level = getNumber<uint8_t>(L, 3);
+		player->sendWorldLight(light);
+		pushBoolean(L, true);
+	} else {
+		lua_pushnil(L);
+	}
+	return 1;
+}
+
+int LuaScriptInterface::luaPlayerSendWorldTime(lua_State* L)
+{
+	// player:sendWorldTime(time)
+	Player* player = getUserdata<Player>(L, 1);
+	if (player) {
+		player->sendWorldTime(getNumber<int16_t>(L, 2));
+		pushBoolean(L, true);
+	} else {
+		lua_pushnil(L);
+	}
+	return 1;
+}
+
```

Comment:
Get rid of those, send the packet using `NetworkMessage` from Lua instead

Reply:
Moved